src/styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,49 @@
+/**
+ * Sass Mixins
+ */
+
+@mixin clearfix {
+	&::after {
+		content: '';
+		display: table;
+		clear: both;
+	}
+}
+
+/*
+$breakpoint arguement choices:
+- phone
+- tab-port
+- tab-land
+- big-desktop
+
+ORDER: Base + typography > general layout + grid > page layout > components
+
+1em = 16px
+*/
+
+@mixin respond($breakpoint) {
+	@if $breakpoint == phone {
+		@media (max-width: 37.5em) {
+			@content;
+		} //600px
+	}
+
+	@if $breakpoint == tab-port {
```

Comment:
Should probably rename to `tab-port-up` for better clarification. Similar with 3 vars bellow

Reply:
what is 'up'? Since we always calculate breakpoint base on screen width, I think keep it short is better